### PR TITLE
Pal 641 create notices file

### PR DESCRIPTION
**Pull request by pd104923**

### CODE REVIEW

File: audit-service-integration/pom.xml [hunk 1]
```diff
@@ -36,6 +39,7 @@
             <groupId>uk.gov.gchq.palisade</groupId>
             <artifactId>audit-service</artifactId>
             <version>${project.parent.version}</version>
+            <classifier>jar-with-dependencies</classifier>
```
Comment (dev930018) [inline audit-service-integration/pom.xml:42]: should have a test scope

File: data-service-integration/pom.xml [hunk 2]
```diff
@@ -42,6 +45,7 @@
             <groupId>uk.gov.gchq.palisade</groupId>
             <artifactId>data-service</artifactId>
             <version>${project.parent.version}</version>
+            <classifier>jar-with-dependencies</classifier>
```
Comment (dev930018) [inline data-service-integration/pom.xml:48]: test scope

File: palisade-service-integration/pom.xml [hunk 3]
```diff
@@ -42,7 +45,7 @@
             <groupId>${project.parent.groupId}</groupId>
             <artifactId>palisade-service</artifactId>
             <version>${project.parent.version}</version>
-            <scope>test</scope>
+            <classifier>jar-with-dependencies</classifier>
```
Comment (dev930018) [inline palisade-service-integration/pom.xml:48]: should have a test scope

**Comment by pd104923** [comment]
> double-check whether you want jars or jars-with-dependencies for all poms, as well as test scopes

There were differing ways of adding the relevant service as a dependency, surely they should be done in the same way? The way I have it is the same as the policy-service, resource-service and user-service modules
It would probably be best to have the test scope on the dependency and not have the `jar-with-dependencies` now I've thought about it.

**Pull request merged**